first try (#1056)
akeneo/pim-helpcenter: content/md/first-steps/manage-multidimensional-data-in-a-table.md

## Import/export the table values of a product

> Exactly like for other attribute types, you can import and export your products with table attributes. But, as a table is a quite complexe value, the only way for the PIM to import table data among the other attributes of the product, is by translating your value into JSON in the "table" attribute column of your CSV or XLSX file.

Reviewer: ```suggestion
Exactly like for other attribute types, you can import and export your products with table attributes. CSV and XLSX formats are structured as tables. Therefore, to import the Akeneo PIM table attribute value, it must be formatted in a single cell of your import using the JSON format. This way it’ll be imported along with any other regular attribute values. It’s faster but more technical. 

```

> ### Use the dedicated "product data table values" import/export jobs

Reviewer: ```suggestion
### Using "product data table values" import/export jobs
```

> We know that, working with JSON format within a xlsx/csv file is not that much comfortable for manual enrichment. That's why we have created several import and export jobs in order to import and export the enriched table attributes, formatted as they are in the product edit form.  

Reviewer 2: ```suggestion
We know that working with JSON format within a xlsx/csv file is not that much comfortable for manual enrichment. That's why we have created several import and export jobs in order to import and export the enriched table attributes, formatted as they are in the product edit form.  
```

Reviewer: ```suggestion
There is another way, more readable and easier to configure using dedicated export and import jobs for the table attribute. It will only handle table attribute values and it will be formatted as they are in the product edit form.  
```

> These dedicated job are named:

Reviewer: ```suggestion
These dedicated jobs are:
```

> - `product data table values` import and export (csv and xlsx)

Author: ```suggestion
- `Product table values` import and export (csv and xlsx)
```

> - `product model data table values` import and export (csv and xlsx)

Author: ```suggestion
- `Product model table values` import and export (csv and xlsx)
```

> You should create **1 import profile per table attribute** you want to import/export. Indeed, these jobs enable you to import/export the table values of several products sharing the same table attribute. As each table attribute can have a different structure, we cannot mix them up in the same file :wink:.

Reviewer 3: In fact they can export values only attached to 1 table attribute, but they can import a file with different attributes. Maybe it's a very specific usecase that don't deserve to be documented...

Reviewer: ```suggestion
We recommend you to create **one import profile per table attribute** you want to import/export. Indeed, these jobs enable you to import/export the table values of several products sharing the same table attribute. For imports, as each table attribute can have a different structure, we don’t want to import files mixing up tables with different structures. For exports, you need to choose a single attribute for each job.
```
